# FrontWheelOdometry.msg - 前车轮里程计反馈
# 轮式里程计信息
float64 left_wheel_odom  # 左轮里程计数
float64 right_wheel_odom # 右轮里程计数